scss/index.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  "extends": [
+    "../css",
+    "stylelint-config-recommended-scss"
+  ],
+  "rules": {
+    "no-descending-specificity": null,
```

Comment:
I had to add this line, otherwise stylelint triggered an error. Not sure why, because I extend `../css`.

Reply:
Fixed.